### DISCUSSION TITLE
New LibriTTS-R voice could replace the current LibriTTS voice

### DISCUSSION THREAD

**Discussion by StoryHack** [rhasspy/piper#184] [Ideas]
A dataset has recently been released which took all the recordings in the LibriTTS dataset and used a restoration model to remove background hiss / echoes / etc. It's called LibriTTS-R. A little bit of work would need to be done to prep this new dataset, but a new high quality voice could be trained from these improved recordings.  They are available at [http://www.openslr.org/141/](http://www.openslr.org/141/)

However, I'm guessing it'd take quite a bit of GPU time to train such a beast. my piddly RTX 3060 would take forever.

I am curious how long the existing libritts voice took to train and on what hardware. 

Also, does anybody have a good colab notebook for training? I wonder if it might be affordable to pay for the processing units to train a new voice in a reasonable time. I've just never written my own notebook.

**Comment by rmcpantoja** [comment]
HI @StoryHack, 
See the notebooks folder in this repo.

**Reply by StoryHack** [reply]
lol, I'm an idiot. Thank you.

**Comment by StoryHack** [comment]
I see elsewhere that the LibriTTS-R dataset is in fact being trained.

**Comment by synesthesiam** [comment]
Currently on epoch 254 :slightly_smiling_face:
